test/autoCompletion.test.ts

```diff
@@ -479,8 +481,8 @@ describe('Auto Completion Tests', () => {
             },
           },
         });
-        const content = 'scripts:\n  sample: test\n  myOther';
-        const completion = parseSetup(content, 31);
+        const content = 'scripts:\n  sample: test\n  myOth|e|r'; // len: 33, pos: 31
+        const completion = parseSetup(content);
```

Comment:
Where I have changed to the new semantic, I add a comment with the original string length, and the caret position.
Once again, this is just for the maintainer's benefit, so they can quickly understand the test's intent